### DISCUSSION TITLE
New Repository

### DISCUSSION THREAD

**Discussion by sarfrazkhan18** [community/community#135664] [Mobile] [answered]
I can not create a new repository on the github mobile android app.

**Comment by Razano26** [comment]
I'm sorry, but on the mobile app you only can do these things:
- Browse your latest notifications
- Read, react, and reply to Issues and Pull Requests
- Review and merge Pull Requests
- Organize Issues with labels, assignees, projects, and more
- Browse your files and code
- Discover new and trending repositories

**Comment by barisaydogdu** [comment]
You can't create a new repository on your mobile app

**Comment by taki-on** [comment, marked as answer]
@sarfrazkhan18 as the folks here already mentioned, we do not support creating a repo on mobile yet. However, I will create an issue to track the feature request. Thank you very much!
